tmt/steps/provision/podman.py

```diff
@@ -199,7 +199,7 @@ def execute(self, command, **kwargs):
         command = directory + environment + command
         return self.podman(
             ['exec'] + interactive +
-            [self.container or 'dry', 'sh', '-c', command], **kwargs)
```

Comment:
personally, I would only fix this. All other changes are with good intent, but I believe it will just hurt my use case and CoreOS, where you cannot install packages via `dnf/yum`

Reply:
Understood. As agreed on the hacking session, let's add the generic `requires` method to all plugins to allow future extensions if needed but do not require `bash` for now so that we do not break @thrix's use case with non-rpm distro.